sdk/communication/communication-phone-numbers/src/sipRoutingClient.ts

```diff
@@ -299,13 +301,53 @@ export class SipRoutingClient {
     );
   }
 
-  private async getRoutesInternal(options: OperationOptions): Promise<SipTrunkRoute[]> {
-    const config = await this.client.sipRouting.get(options);
+  /**
+   * Test routes with number.
+   * @param targetPhoneNumber - target phone number to test routes.
+   * @param routes - sip trunk routes.
+   * @param options - The options parameters for routes.
+   */
+  public async testRoutesWithNumber(
+    targetPhoneNumber: string,
+    routes: SipTrunkRoute[],
+    options: OperationOptions = {},
+  ): Promise<TestRoutesWithNumberResponse> {
+    return tracingClient.withSpan(
+      "SipRoutingClient-testRoutesWithNumber",
+      options,
+      async (updatedOptions) => {
+        return this.testRoutesWithNumberInternal(targetPhoneNumber, routes, updatedOptions);
+      },
+    );
+  }
+
+  private async testRoutesWithNumberInternal(
+    targetPhoneNumber: string,
+    routes: SipTrunkRoute[],
+    options?: OperationOptions,
+  ): Promise<TestRoutesWithNumberResponse> {
+    const optionalParams = {
+      routes: routes,
+      ...options,
+    };
+    return this.client.sipRouting.testRoutesWithNumber(targetPhoneNumber, optionalParams);
+  }
+
+  private async getRoutesInternal(options: SipRoutingGetOptions): Promise<SipTrunkRoute[]> {
+    const optionalParams = {
+      expand: options.expand !== undefined ? options.expand : undefined,
```

Comment:
For getRoutes, we can always call without the expand parameter, because routes don't carry any health information.